src/Sylius/Bundle/ShopBundle/Resources/views/Account/requestPasswordReset.html.twig

```diff
@@ -17,12 +17,14 @@
             </h1>
             <div class="ui segment">
                 {{ form_start(form, {'action': path('sylius_shop_request_password_reset_token'), 'attr': {'class': 'ui large loadable form', 'novalidate': 'novalidate'}}) }}
-                {{ form_row(form.email) }}
+                {{ form_row(form.email, sylius_data_test_attribute_array('reset-email')) }}
```

Comment:
```suggestion
                {{ form_row(form.email, sylius_test_form_attribute('reset-email')) }}
```
It's an array, but it's highly specific for forms and should only be used there - hence my proposition to change the name.